site/docs/projection.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+menu: docs
+title: Projection
+permalink: /docs/projection.html
+---
+## Projection Overview
+Projection's are specified at the mark specification level, alongside encoding.
```

Comment:
I think we usually begin a docs by explaining the concept. You can copy some of the text from  https://vega.github.io/vega/docs/projections

> specified at the mark specification level, alongside encoding

Also, there is no concept of "mark specification".  I think you only mean specification.  Please also make it clear whether all types of specification support `projection`.  (Is it only unit and layer or all types of specification?)